lib/stdlib/doc/src/gen_server.xml

```diff
@@ -99,6 +99,11 @@ gen_server:abcast     -----> Module:handle_cast/2
       in place of the time-out or hibernation value, which will immediately
       invoke the <c>handle_continue/2</c> callback.</p>
 
+    <p>If the <c>gen_server</c> process terminates with any reason other
+      than <c>normal</c>, e.g. as a result of a function in the callback
+      module returning <c>{stop,Reason,NewState}</c> with a reason other
+      than <c>normal</c>, an exit signal is emitted.</p>
```

Comment:
I'm pretty sure you get a `normal` exit signal regardless and that you can see it with `trap_exit`, otherwise supervisors would have a hard time

Reply:
Ah, right! I'll change this and other mentions of "normal".

Reply:
Ah, you're right. I wanted to mention that the exit signal is the same as Reason in various places. I'll revert all mentions of "normal" in this PR.